content/services/service_codegov.md

```diff
@@ -23,9 +23,8 @@ weight: 2
 
 # Topics that best describe this product or service
 topics:
-  - open-source
-  - code
```

Comment:
I think we should keep the topics the same. They directly affect how they are displayed in the [resource](http://localhost:1313/topics/) listing. 

The `code` topic gets a more prominent page:
https://digital.gov/topics/code/

Same with `open-source`:
http://localhost:1313/topics/open-source/

While the lesser-used topics (like federal source code and open source software) would get a basic page like:
https://digital.gov/topics/fitara/

Reply:
I don't like your topics. They do adequately describe Code.gov.

Want me to create an issue to update you topics? ;-)